.devcontainer/cuda/devcontainer.json

```
@@ -1,8 +1,8 @@
 {
-  "name": "Autoware-cuda",
+  "name": "autoware:universe-cuda",
```

I think it is better to understand. 

```suggestion
  "name": "autoware:universe-devel-cuda",
```